DigitalLearningSolutions.Data/Services/ActionPlanService.cs

```diff
@@ -150,5 +150,28 @@ public async Task<IEnumerable<ActionPlanItem>> GetIncompleteActionPlanItems(int
 
             return resource.Link;
         }
+
+        public void RemoveActionPlanItem(int learningLogItemId, int delegateId)
+        {
+            var removalDate = clockService.UtcNow;
+            learningLogItemsDataService.RemoveLearningLogItem(learningLogItemId, delegateId, removalDate);
+        }
+
+        public bool? VerifyDelegateCanAccessActionPlanItem(int learningLogItemId, int delegateId)
+        {
+            if (!config.IsSignpostingUsed())

```

Comment:
So is it only resources that go in the LearningLog? I thought other things like courses and self-assessments did too.

Reply:
Or are you making a distinction between LearningLog and ActionPlan?

Reply:
Self assessments and courses have their progress saved in other tables than LearningLogItems. Kevin did mention that there was one centre that still used LearningLogItems, but cannot recall what exactly it was used for in that case. Whatever that old case was, I doubt we will need to be modifying it at all as it is/was being phased out.

Either way, I've not seen the terminology of "action plan" used outside of the context of learning resources - from what I can tell, courses/self assessments can be enrolled on by the user from the "available" activities page, while learning resources will have to explicitly be "added to action plan" from the Recommended Learning page (at least for now).

Reply:
Okay, sounds reasonable if ActionPlan is inherently linked to Signposting, though I think this only makes sense for resources? If so, I'd change the method name `Item` -> `Resource`

Reply:
I've now changed the method name, as well as the name of the identically named service filter that called this. Do we think the model "ActionPlanItem" should be renamed similarly as well?

Reply:
It has fields like ResourceType and ResourceLink, so yeah, I think so.

Reply:
This has now been renamed, along with all the places it was used. This might end up causing some merge conflicts with other tickets, so I'll comment on it in the slack channel and will keep and eye out for any merges that happen in the mean time.